Roadmap.md

```diff
@@ -0,0 +1,81 @@
+## Short-term
+
+🚧 Review new contributions (high priority)
+
+- There are usually 10-15 open PRs going stale which is inefficient because they require plenty of work to update (when there's conflicts) and sometimes they get duplicated like [this](https://github.com/aragon/aragon-cli/pull/254) and [this](https://github.com/aragon/aragon-cli/pull/272)
+
+🚧 [Implement a mono repo structure with `lerna`](https://github.com/aragon/aragon-cli/pull/325) 
+
+🚧 Finish the `aragon start` command [here](https://github.com/aragon/aragon-cli/pull/255/files) which is intended to be more development friendly that `aragon run`, i.e. [doesn't create a new dao every time](https://github.com/aragon/aragon-cli/issues/311), uses http rather than ipfs (to have hot/live reload for the frontends, and maybe something similar for the contracts, a watch → compile loop) 
+
+👾 [E2E tests with sharness](https://github.com/aragon/aragon-cli/issues/358)
+
+👾 Bugfixes, see all [here](https://github.com/aragon/aragon-cli/labels/bug)
+
+- [gasLimit](https://github.com/aragon/aragon-cli/issues/346) issue
+- [gas issue on dao acl grant](https://github.com/aragon/aragon-cli/issues/350)
+- [dao new fails with ganache 2.3](https://github.com/aragon/aragon-cli/issues/321) (low priority because pinning works alright at the moment)
+
+Note: We should write tests as we fix these to ensure no regressions!
+
+👾 Small features (low hanging fruit)
+
+- remove [hard-coded gas price](https://github.com/aragon/aragon-cli/issues/353) (would be nice to let the user decide how much)
+
+👾 [Claiming an aragonid](https://github.com/aragon/aragon-cli/issues/347) 
+
+👾 Document new features (dao token commands)
+
+👾 Restructure documentation on hack.aragon to include:
+
+- Quick start / Overview / Introduction
+- Guides around specific subjects (e.g.: Publish to different environments, Using a different Ethereum account, Propagating content, etc), similar to [this](https://github.com/aragon/aragon-react-boilerplate#publish). Maybe also link more in-depth guides such as [this](https://forum.aragon.org/t/guide-custom-aragon-organization-deployment-using-the-cli/507)
+- FAQ section. We have [this](https://hack.aragon.org/docs/cli-usage.html#troubleshooting-faq) currently, but it should be it's own section and much more elaborate (If there is a recurring question on #dev-help we will add it there)
+
+## Long-term
+
+- Types (use Flow or Typescript) because it will allow us to catch more "errors" at compile time, having commands silently fail without noticing (like [this](https://github.com/aragon/aragon-cli/pull/334#discussion_r248659171)) less often. Using Typescript is also great for intellisense support (having auto-completion from IDEs) and refactoring
+- Improve smart contract tooling using new Ethereum dev tools from 0x:
```

Comment:
**Brett:** I wouldn't say these are necessarily within the scope of the CLI, but we can explore if replacing truffle makes sense in the CLI (probably doesn't, but we can upgrade to v5) and solidity-coverage in the boilerplates (really hoping this is possible).

Reply:
I Been researching about this and the tools are framework agnostic so they can be configure with truffle if we want to keep using it. Here is a basic example: https://github.com/0xProject/dev-tools-truffle-example